Refresh.GameServer/Extensions/LevelEnumerableExtensions.cs

```diff
@@ -38,6 +38,9 @@ public static IQueryable<GameLevel> FilterByLevelFilterSettings(this IQueryable<
         if (levelFilterSettings.ExcludeMyLevels && user != null)
             levels = levels.Where(l => l.Publisher != user);
 
+        if (user is { ShowModdedContent: false })
+            levels = levels.Where(l => !l.Modded);
+
```

Comment:
We'll probably want to also have an option in `levelFilterSettings` for a temporary override when browsing on the website.